.github/workflows/release.yaml

```diff
@@ -20,21 +23,124 @@ jobs:
         with:
           go-version: '1.22'
 
+      - name: Build and Package Go executable
+        run: |
+          cd src
+          go mod tidy
+          GOOS=linux GOARCH=amd64 go build -o bootstrap main.go
+          zip -r ../$log_forwarder_zip_file_name .
+          cd ..
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: log-forwarder-zip
+          path: |
+            ${{ env.log_forwarder_zip_file_name }}
+
+      - name: Install AWS SAM CLI
+        run: |
+          pip install aws-sam-cli
+
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v3
         with:
-          role-to-assume: ${{ secrets.AWS_SAR_PUBLISHER_ROLE }}
+          role-to-assume: ${{ secrets.AWS_S3_PUBLISH_ROLE }}
           aws-region: us-east-2
 
-      - name: Install AWS SAM CLI
+      - name: Build and Package SAM Application and Upload to S3
```

Comment:
Shall we first deploy the lambda code to all region and then build and deploy the templates ?

Reply:
We can do that, I actually separated lambda deployment part so it can be done parallely to all regions. Hence I moved it out as a separate job. If template deployment have to happen after lambda deployment then template deployment has to be moved to another job which will make CI/CD complicated. I don't see any issue if template gets uploaded before lambda code. Let me know what you think

Reply:
@hrai-nr I have separated the files for lambda deployment and template deployment so we can run them only when specific changes are there. So there's no way we add a condition that template deployment happens only after lambda deployment